plugins/chat/assets/javascripts/discourse/components/chat-message-thread-indicator.hbs

```diff
@@ -3,11 +3,41 @@
   @models={{@message.thread.routeModels}}
   class="chat-message-thread-indicator"
 >
-  <span class="chat-message-thread-indicator__replies-count">
-    {{i18n "chat.thread.replies" count=@message.threadReplyCount}}
-  </span>
-  <span class="chat-message-thread-indicator__view-thread overflow-ellipsis">
-    {{i18n "chat.thread.view_thread"}}{{#if this.threadTitle}}:
-      {{replace-emoji this.threadTitle}}{{/if}}
-  </span>
+  <div class="chat-message-thread-indicator__last-reply-avatar">
+    <ChatUserAvatar
+      @user={{@message.thread.preview.lastReplyUser}}
+      @avatarSize="small"
+    />
+  </div>
+  <div class="chat-message-thread-indicator__body">
+    <div class="chat-message-thread-indicator__last-reply-metadata">
+      <span class="chat-message-thread-indicator__last-reply-username">
+        {{#if this.site.mobileView}}
+          <div class="chat-message-thread-indicator__last-reply-avatar -mobile">
+            <ChatUserAvatar
+              @user={{@message.thread.preview.lastReplyUser}}
+              @avatarSize="small"
+            />
+          </div>
+        {{/if}}
+        {{@message.thread.preview.lastReplyUser.username}}
```

Comment:
I would prefer if it had a direct overlapping span with a class

Reply:
Not sure what you mean in this case it is already in a span

Reply:
```
<span>
  foo
  <span>bar</a>
</span>
```

In in this case foo has no direct enclosing element:

```
<span>
  <span>foo</span>
  <span>bar</a>
</span>
```

It's not always necessary, but it's easier to reason about as you can have alignment issues otherwise and easier to style if necessary.

Reply:
Ah yep I see, will fix :+1: